package-lock.json

```diff
@@ -759,6 +759,13 @@
         "foundationdb": "^2.0.1"
       }
     },
+    "node_modules/@dbos-inc/dbos-sdk/node_modules/reflect-metadata": {
```

Comment:
Remove this?

Reply:
It's an artifact of the peer dependencies in our monorepo, not real. We'll clean up the peer dependencies later.